### ISSUE TITLE
2.3 Compatibility Chat Card Display

### ISSUE THREAD

**Issue by Neurochimp** [lipefl/swade-tools#215]
Hi.

2.3 went live & so far only seeing a minor issue [fingers crossed] with dice being displayed in the chat card like as follows: {d10x, d6x}kh.
![d10kh](https://user-images.githubusercontent.com/116909768/232172601-f9a10e1a-c539-4f71-95e2-09b5f173f68b.png)

**Comment by inxaos** [comment]
The buttons also seem a little off in your screenshot and there similar "alignment" related issues on the skill check card as well.

**Comment by lipefl** [comment]
Well, i just updated to test and i can't even open an actor sheet to test (even disabling all modules and creating a new world).

**Comment by inxaos** [comment]
Are you using an electron install or web? I'm all web (I have a local dedicated server) and haven't had any problems.

**Comment by lipefl** [comment]
I tested only on PC. I'm afraid to update it on web server right now. I'll look at discord.

Btw, @inxaos did you follow any tutorial install foundry on a dedicated server? I have one at my disposal, but i'm not sure yet how to do it (i actually use google cloud).

**Comment by inxaos** [comment]
> I tested only on PC. I'm afraid to update it on web server right now. I'll look at discord.
> 
> Btw, @inxaos did you follow any tutorial install foundry on a dedicated server? I have one at my disposal, but i'm not sure yet how to do it (i actually use google cloud).

I have a server rack in my garage I run kubernetes on, so I just run the published docker image there. That being said, I'd be happy to help figure out how to do the equivalent in GCE.

**Comment by lipefl** [comment]
I did it following a tutorial (in portuguese), it's not that hard. There's probably some english ones on youtube.

**Comment by Alis-72** [comment]
This seem to have been fixed in SWADE 2.3.4

**Comment by lipefl** [comment]
The "formula" showing is coming from SWADE system. It's affecting basic rolls of the system (that SWADE Tools uses for rolls). I've warned them and they'll fix it.
I'll leave the issue open if I need to come back to it later.

**Comment by Neurochimp** [comment]
![Frenzy_Issue](https://user-images.githubusercontent.com/116909768/235268854-212b0812-14b7-44a5-adf7-07f5c54a6213.png)
Been sick the last week, so last night was the first real game I ran.
Frenzy looks like it's being added together on the new card, but still assessed separately below. One of my players was confused as he thought the total was a hit but the dice split between attacks was misses.

I took a look at the ticket I think you opened over on the core rules Github. Looks like they're resisting addressing this issue due to so much code that would have to be rewritten.
